src/pyosmeta/cli/process_reviews.py

```diff
@@ -35,27 +34,21 @@ def main():
     # Get all issues for approved packages - load as dict
     # TODO: this doesn't have to be in process issues at all. it could fully
     # Call the github module
-    issues = process_review.return_response()
-    accepted_reviews = process_review.parse_issue_header(issues, 45)
+    issues = process_review.get_issues()
```

Comment:
Above - if you rebase from main (i am pretty sure_ i fiex this

`labels=["6/pyOS-approved 🚀🚀🚀"]` -->
labels=["6/pyOS-approved"]

this is why we had a magical deleted file in our last pr 🙈  i removed the emojis.

Reply:
merged the new label in main!

Reply:
thank you!! now we are no longer deleting all of the packages ✨  😆